EosGround/db_setup.py

```diff
@@ -0,0 +1,44 @@
+import psycopg2
+from config.config import config
+
+conn_params = config('database.ini')
+connection = psycopg2.connect(**conn_params)
+connection.set_isolation_level(psycopg2.extensions.ISOLATION_LEVEL_AUTOCOMMIT)
+cur = connection.cursor()
+
+try:
+    cur.execute(
+        """
+        CREATE TABLE receive_table (
+        id SERIAL PRIMARY KEY,
+        packet_type INT NOT NULL,
+        packet_sender INT NOT NULL,
+        packet_priority INT NOT NULL,
+        packet_generate_time timestamp NOT NULL,
+        packet_sequence_number INT NOT NULL,
+        packet_timestamp timestamp NOT NULL,
+        packet_body VARCHAR(255) NOT NULL,
+        time_arrived timestamp NOT NULL
+        )
+        """
+    )
+except:
+    print("RX table exists")
```

Comment:
This except block is not necessarily accurate -- for example, incorrect creds would throw.  You should catch a specific exception type